downstream/modules/platform/con-controller-inventory-sync-jobs.adoc

```diff
@@ -0,0 +1,22 @@
+[id="controller-inventory-sync-jobs"]
+
+= Inventory sync jobs
+
+When an inventory synchronization is executed, the full results display in the *Output* tab. 
+This shows the same information if you run it through the Ansible command line, and can be useful for debugging. 
```

Comment:
If used, the Ansible CLI displays the same information. This can be useful for debuugging.